pkg/supervisor/supervisor_windows.go

```diff
@@ -0,0 +1,32 @@
+//go:build windows
+// +build windows
+
+/*
+Copyright 2022 k0s authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package supervisor
+
+import (
+	"time"
+)
+
+// maybeKillPidFile checks kills the process in the pidFile if it's has
+// the same binary as the supervisor's. This function does not delete
+// the old pidFile as this is done by the caller.
+func (s *Supervisor) maybeKillPidFile(check <-chan time.Time, deadline <-chan time.Time) error {
+	s.log.Warnf("maybeKillPidFile is not implemented on Windows")
```

Comment:
I wonder why that is. There's still the possibility of sending `os.Kill` to terminate the process. Anyhow, we can address this later on.

Reply:
I think it's worth merging this as is and merging your patch for this for several reasons:
1- I don't have a windows environment to properly test this and we need to get this merged ASAP because of the release.
2- Windows is broken at the moment anyway.
3- Your solution is almost implemented right now and it's a good implementation. Duplicating the effort seems unreasonable to me.

Anyway we need a way to get windows workers for our local environments.